From 0874e5fb0a2e62e0d3e5c77ba6a888f786e95b32 Mon Sep 17 00:00:00 2001
From: Evan Goetz <evan.goetz@ligo.org>
Date: Tue, 29 Apr 2025 15:54:59 -0700
Subject: [PATCH] pyproject.toml: remove lalsuite from dependencies

the component libraries we actually need are specified individually in the conda recipe
---
 pyproject.toml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index bb3ffc7..274d1f8 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -42,7 +42,6 @@ dependencies = [
   "gwdetchar >=2.2.7",
   "gwpy >=3.0.9",
   "gwtrigfind",
-  "lalsuite",
   "lscsoft-glue >=1.60.0",
   "lxml",
   "markdown",
-- 
2.49.0

